app/tests/unit/pages/nanomine/howto.spec.js

```diff
@@ -1,35 +1,35 @@
-import { mount, createLocalVue } from '@vue/test-utils'
-import Vuex from 'vuex'
-
+import createWrapper from '../../../jest/script/wrapper'
 import HowTo from '@/pages/nanomine/howTo/HowTo.vue'
-import store from '@/store'
 
-const localVue = createLocalVue()
-localVue.use(Vuex)
+var wrapper = null
+// suppress jsdom alert 'Not implemented window.open'
+window.open = jest.fn()
+window.open.mockClear()
+global.console = {
+  log: jest.fn(), // console.log are ignored in tests
 
-beforeAll(() => {
-  jest.resetModules()
-})
+  // Keep native behavior for other methods
+  error: console.error,
+  warn: console.warn,
+  info: console.info,
+  debug: console.debug
+}
 
 describe('HowTo.vue', () => {
+  beforeAll(async () => {
+    wrapper = createWrapper(HowTo, { props: {}, slots: {} })
```

Comment:
I will suggest if the props and slots are empty, we can throw in an empty object like so: 
`createWrapper(HowTo, {})`

Reply:
Done in ca97924